uptane/services/timeserver.py

```diff
@@ -19,7 +19,7 @@
 try:
  import pyasn1.type
 except ImportError:
- print('Minor: pyasn1 library not found. Proceeding using JSON only.')
+ uptane.logger.error('pyasn1 library not found. Proceeding using JSON only.')
```

Comment:
👍 
I verified that print() is not used by the other modules in the reference implementation, and only found its use in the demo code.